_includes/shared/markup/ccloud/append-ccloud-config.adoc

```diff
@@ -0,0 +1,2 @@
+Append the contents of the `configuration/ccloud.properties` to `configuration/dev.properties`
```

Comment:
Show the `cat` command to do this

Reply:
slick, added in commit https://github.com/confluentinc/kafka-tutorials/pull/786/commits/71cceceb46a091134d731263f01929e75b9e687c